backend/app/infrastructure/database/models.py

```diff
@@ -52,6 +52,14 @@ class GPStamps(BaseModel):
     stamps = Column(db.String, nullable=False)
 
 
+class HolonymSBT(BaseModel):
```

Comment:
i also started thinking if we shouldn't match HolonymSBT with a specific epoch that it corresponds to? it'll help to distinguish between these tokens that are valid and expired ones (since we want to verify it for every epoch specifically, right?)